graphql/context.go

```diff
@@ -195,3 +195,55 @@ func (c *RequestContext) RegisterExtension(key string, value interface{}) error
 	c.Extensions[key] = value
 	return nil
 }
+
+const (
+	initpayload key = "initpayload_context"
+)
+
+// InitPayload is a structure that is parsed from the websocket init message payload. TO use
+// request headers for non-websocket, instead wrap the graphql handler in a middleware.
+type InitPayload map[string]interface{}
```

Comment:
Does leaving this in the `handler` package cause an import loop? It doesn't feel right in here, surrounded by other graphql transport agnostic stuff.

Reply:
I'll look into it.